package.json

```diff
@@ -26,6 +26,8 @@
     "@react-navigation/native": "6.1.7",
     "@react-navigation/stack": "6.3.17",
     "@sentry/react-native": "^5.6.0",
+    "@walletconnect/core": "^2.7.6",
+    "@walletconnect/web3wallet": "^1.4.0",
```

Comment:
Why do you need this here? This should be added only if the patch is applied, otherwise a simple `npm install` would install them.

Reply:
You're right, I'm not sure how this got there, fixed